### DISCUSSION TITLE
new to this

### DISCUSSION THREAD

**Discussion by anonymoususerhere** [thatskymod/Sky-CotL-Scripts#124] [Q&A] [answered]
i'm kinda new to this, so how do i extract the stuff to the game directory? i play on PC/Steam

**Comment by thatskymod** [comment, marked as answer]
These resources appear relevant to your question:

1. Automated Installation Instructions:
   - https://github.com/thatskymod/Sky-CotL-Scripts/tree/main/PC#automated-installation-instructions
2. Manual Installation Instructions:
   - https://github.com/thatskymod/Sky-CotL-Scripts/tree/main/PC#manual-installation-instructions

Using the automated installer is generally preferred, as it handles all the extraction steps for you and minimizes the likelihood of errors.

The [README](https://github.com/thatskymod/Sky-CotL-Scripts/tree/main/PC#readme) file has now been updated with a project overview, features, and screenshots to better guide new users.
